ext/puma_http11/mini_ssl.c

```diff
@@ -319,7 +349,7 @@ VALUE engine_inject(VALUE self, VALUE str) {
   return INT2FIX(used);
 }
 
-static VALUE eError;
+//static VALUE eError;
```

Comment:
Can be removed, yes, since you added it higher up?

Reply:
Sorry.  Fixed soon.